### ISSUE TITLE
Exosuit chestplate duping with steam filler

### ISSUE THREAD

**Issue by CurePindal** [Flaxbeard/Flaxbeards-Steam-Power#215]
We noticed that the exosuit chestplate is duped in multiplayer when taken from the steam filler.
Is 100% reproducable on our claudron server..This does not seem to happen in singleplayer.
1. put exo chestplate on steam filler
2. take it off the steam filler
3. 2 items in your inventory. One seem to have the previous steam amount, the other one the charged amount.

**Comment by elifoster** [comment]
I will remove the [wontfix](https://github.com/Flaxbeard/Flaxbeards-Steam-Power/labels/wontfix) label if you can verify that this happens *without* Cauldron installed. Until then, this will *not* be a priority. We don't support Cauldron.

**Comment by CurePindal** [comment]
This issue turned even out more weird. A few mins later all duped chestplates vanished from the players inventory. Either theres some issue with the packets or this is the stranges desync ive encountered so far.
Will test again on a forge server tomorrow.

**Comment by elifoster** [comment]
Yeah that's just a desync issue by the sound of it.

**Comment by CurePindal** [comment]
Yep, akward desync, not a bug!

**Comment by CurePindal** [comment]
I just managed to dupe the chestplate on our cauldron server. And because a lot of servers run cauldron (only way to properly run a public server) this should be considered worthy to be reviewed. Theres so far no other item duping, so this can be a problem with Steam mods packet handling.
Tests on a pure forge server underway.

**Comment by elifoster** [comment]
Until you can show that it happens on a non cauldron server, it won't be priority. Cauldron is not supported.

**Comment by banjomarx** [comment]
i hate to break it to you elif but if you DONT support cauldron 90% of the people playing your mod will have to remove it from their packs, yes ik thats not fair and it sucks, and ik that its pretty much impossible to work with cauldron now as a modder, but its the sad truth that pretty much all modded MC servers use Cauldron.

**Comment by xbony2** [comment]
We don't like or support Cauldron. Deal with it.

**Comment by banjomarx** [comment]
ill deal with it, i already have. by removing your mod from my pack/server. good day

**Comment by elifoster** [comment]
@banjomarx You literally just said that it's impossible to work with Cauldron, so why should we support it?

**Comment by banjomarx** [comment]
its like this, lets say that I have a bad leg and i cant walk well... should i give up and lie around all day? have it chopped off? or do i work on walking each day and get it a lil better everytime. What im saying is no supporting cauldron is effectively chopping off that leg... I dont see why you guys would want to cut off all the people who use it considering that every single server owner ik uses it... well i understand working with it is hard, but it MIGHT be possible that from asking about the problem and looking at it you might be able to figure something out (and yes ik from personal experience as a coder that its usually not that simple). My point is that the refusal to even try to look at the problem will just push away the player base and mean less and less modpacks will use it until sponge comes out, because when its a major dupe like this, its just not something you can work around on a server.
Im not asking for miracles, im just asking not to get a door in the face and maybe a few questions. the problem is a dupe bug meaning you can pretty clearly identify what parts of the code are going to be related, especially if you can confirm that its not happening with other items

**Comment by elifoster** [comment]
Well, its not even a bug that can be fixed,  since its likely just desync. If not examining an issue that probably not our fault some people stop using the mod, then those people are honestly ridiculous.

**Comment by banjomarx** [comment]
its not about whether its "your fault" if the mod isnt usable theres nothing we can do.... and its not usable if the main point of the mod afaik (the armr) is easily dupeable. and since it doesnt happen with mods like TE and stuff with their coding their must be some difference in your coding allowing the dupe to happen. that doesnt mean that you are a bad coder or w/e it just means that the coding is different but as long as this easy dupe continues server owners will have NO CHOICE but to either remove the armr or remove the mod, which isnt much of a choice...

**Comment by elifoster** [comment]
I dont think you understand what im saying. If you read the thread you would have seen that the dupe isnt an actual dupe

**Comment by banjomarx** [comment]
but then cue reopened it cuz it WAS a dupe

**Comment by banjomarx** [comment]
its been done and confirmed on my server as well

**Comment by banjomarx** [comment]
sorry i guess we werent really understanding eachother,, it is a dupe, it even gets picked up by server plugins like bukkit /seeinv etc. and on my server atleast it didnt disapear

**Comment by ProsperCraft** [comment]
This is a dupe, running a cauldron server as well. 
I made a video for you

https://youtu.be/DbVAfGsIeZU

I just now made 7, relogged and still had 7.

Put them in a chest, removed them from a chest, still 7.

Dropped them on the ground and picked them up, 30 minutes later still have them.

Version - 1.7.10-0.28.7

**Comment by elifoster** [comment]
No one ever actually answered an important question: Does this happen without Cauldron? On a "Vanilla" server

**Comment by Drullkus** [comment]
This was an entertaining read
